doc/api/events.markdown

```diff
@@ -164,7 +164,7 @@ Returns emitter, so calls can be chained.
 Removes a listener from the listener array for the specified event.
 **Caution**: changes array indices in the listener array behind the listener.
 
-    var callback = function(stream) {
+    const callback = function(stream) {
       console.log('someone connected!');
     };
```

Comment:
Maybe just use a function... definition? here. (I forget if that's the correct term or not...)